specs/RasterizationScale.md

```diff
@@ -0,0 +1,164 @@
+        eventHandler,
+    [out] EventRegistrationToken* token);
+  /// Remove an event handler previously added with
+  /// add_RasterizationScaleChanged.
+  HRESULT remove_RasterizationScaleChanged(
+    [in] EventRegistrationToken token);
+}
+interface ICoreWebView2RasterizationScaleChangedEventHandler : IUnknown {
+  /// Called to provide the implementer with the event args for the
+  /// corresponding event. There are no event args and the args
+  /// parameter will be null.
+  HRESULT Invoke(
+    [in] ICoreWebView2Controller2* sender,
+    [in] IUnknown* args);
+}
+```
+## .Net and WinRT
+```c#
+namespace Microsoft.Web.WebView2.Core
+{
+public partial class CoreWebView2Controller
+    {
+        /// <summary>
+        /// Gets or sets the WebView rasterization scale.
+        /// </summary>
+        /// <remarks>
+        /// The rasterization scale is the combination of the monitor DPI scale and text scaling set by the user. This value shoud be updated when the DPI scale of the app's top level window changes (i.e. monitor DPI scale changes or the window changes monitor) or when the text scale factor of the system changes.
+        /// Rasterization scale applies to the WebView content, as well as popups, context menus, scroll bars, and so on. Normal app scaling scenarios should use the <see cref="CoreWebView2.ZoomFactor"/> property or <see cref="CoreWebView2.SetBoundsAndZoomFactor"/> method.
+        /// </remarks>
+        public double RasterizationScale { get; set;};
+
+        /// <summary>
+        /// Determines whether the WebView will detect monitor scale changes.
+        /// </summary>
+        /// <remarks>
+        /// ShouldDetectMonitorScaleChanges property determines whether the WebView attempts to track monitor DPI scale schanges. When true, the WebView will track monitor DPI scale changes, update the <see cref="CoreWebView2.RasterizationScale"/> property, and fire <see cref="CoreWebView2.RasterizationScaleChanged"/> event. When false, the WebView will not track monitor DPI scale changes, and the app must update the <see cref="CoreWebView2.RasterizationScale"/> property itself. <see cref="CoreWebView2.RasterizationScaleChanged"/> event will never fire when ShouldDetectMonitorScaleChanges is false.
+        /// </remarks>
+        public bool ShouldDetectMonitorScaleChanges {get; set;};
+
+        /// <summary>
+        /// RasterizationScalechanged is raised when the <see cref="CoreWebView2Controller.RasterizationScale"/> property changes.
+        /// </summary>
+        /// <remarks>
+        /// The event is raised when the Webview detects that the monitor DPI scale has changed, <see cref="CoreWebView2Controller.ShouldDetectMonitorScaleChanges"/> is true, and the Webview has changed the <see cref="CoreWebView2Controller.RasterizationScale"/> property.
+        /// </remarks>
+        /// <seealso cref="CoreWebView2Controller.RasterizationScale"/>
+        public event EventHandler<object> RasterizationScaleChanged;
```

Comment:
TypedEventHandler?

Reply:
It should be a TypedEventHandler with a typed sender and typed args, and guideline is for typed args, even though there's no members (and then pass null at runtime).